tcl/org.testeditor.tcl.dsl/src/org/testeditor/tcl/dsl/jvmmodel/TclJvmModelInferrer.xtend

```diff
@@ -213,249 +229,254 @@ class TclJvmModelInferrer extends AbstractModelInferrer {
 
 	private def void generateEnvironmentVariableAssertion(EnvironmentVariable environmentVariable,
 		ITreeAppendable output) {
-		output.append('''org.junit.Assert.assertNotNull(«expressionBuilder.variableToVarName(environmentVariable)»);''').
+		output.
+			append('''org.junit.Assert.assertNotNull(«expressionBuilder.variableToVarName(environmentVariable)»);''').
 			newLine
 	}
 
-	private def void generate(SpecificationStepImplementation step, ITreeAppendable output,
-		Iterable<EnvironmentVariable> environmentVariables) {
-		val comment = '''/* «step.contents.restoreString» */'''
-		output.newLine
-		output.append(comment).newLine
-		step.contexts.forEach[generateContext(output.trace(it), #[], environmentVariables)]
-	}
+			private def void generate(SpecificationStepImplementation step, ITreeAppendable output,
```

Comment:
Unfortunately, the Xtend formatter does not like this file :( could you please revert the auto-formatting?